frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -128,15 +137,13 @@ export function Experiment(): JSX.Element {
                                             placeholder="Adding a helpful description can ensure others know what this experiment is about."
                                         />
                                     </Form.Item>
-                                    <Form.Item
-                                        label="Select participants"
-                                        name="person-selection"
-                                        className="person-selection"
-                                    >
+                                </Col>
+                                <Col span={12}>
+                                    <Form.Item label="Select participants" name="person-selection">
                                         <Col>
                                             <div className="text-muted">
-                                                Select the entities who will participate in this experiment. We'll split
-                                                all participants evenly into a <b>control</b> and <b>test</b> group.{' '}
+                                                Select the entities who will participate in this experiment.
```

Comment:
```suggestion
                                                Select the entities who will participate in this experiment. By default, all persons are selected.
```

Reply:
Chris has a better version in the mocks: basically saying that no filter means everyone is selected.

Reply:
ahh ok